.github/workflows/integration_test_dolphin.yml

```diff
@@ -258,8 +258,10 @@ jobs:
           cd ${{ github.workspace }}/Manta/tests
           yarn install
           yarn
-          yarn correctness_test --address=ws://127.0.0.1:9921 --exit
-          yarn performance_test --address=ws://127.0.0.1:9921 --exit
+          yarn init-assets
+          yarn mint-private-assets
+          yarn validate-pull-ledger-diff
+          yarn performance_test
```

Comment:
Why is the address removed ?

Reply:
Aha I see you made it the default, but I'd prefer if we still have the caller pass it through, so it is more explicit and different places will need different addresses anywa.

Reply:
The argument `address` is not removed, you still can run it like before
```
yarn runtime_upgrade_test --address=ws://127.0.0.1:9921 --exit
```

If you don't specify `address`, the script will get it from `.github/resources/config-for-runtime-upgrade-test.json`

Reply:
I know, i just think it is clearer to pass it as a parameter explicitly, but up to you.